src/distance/vectorize.hpp

```diff
@@ -34,15 +33,15 @@ class VectorGenerationAlgorithm {
  * 
 */
 class LOSTVectorGenerationAlgorithm : public VectorGenerationAlgorithm {
-public:
-
+ public:
     /**
      * Creates a LOSTVectorGenerationAlgorithm object
      * 
      * @param orientation The orientation of the satellite as determined by LOST
     */
-    LOSTVectorGenerationAlgorithm(Vec3 orientation/*Params to initialze fields for this object*/) : orientation(orientation) {};
-    
+    explicit LOSTVectorGenerationAlgorithm(Vec3 orientation/*Params to initialze fields for this object*/)
```

Comment:
nit: comment inside paren looks a bit weird to me, though I've never seen this before, what's the reasoning?

Reply:
Oh, just to warn people about what they need to feed into here, but its going to change soon, so I won't edit it now